packages/debug/src/browser/debug-session.tsx

```diff
@@ -759,4 +741,27 @@ export class DebugSession implements CompositeTreeElement {
         return this.threads;
     }
 
+    protected handleContinued = ({ body: { allThreadsContinued, threadId } }: DebugProtocol.ContinuedEvent): void => {
```

Comment:
From [here](https://github.com/eclipse-theia/theia/issues/8614#issuecomment-706283155):
> only that good practice that doesn't hurt anything upstream

Let's make it return with `Promise<void>` so downstreams have greater flexibility on the code customization. Same for `handleThread`. What do you think?

Reply:
Sounds fair to me - I'll make the change.